src/ucs/arch/cpu.h

```diff
@@ -68,6 +68,12 @@ typedef enum ucs_cpu_cache_type {
     UCS_CPU_CACHE_LAST
 } ucs_cpu_cache_type_t;
 
+/* Built-in memcpy type of value */
+typedef enum ucs_cpu_builtin_memcpy_val {
+    UCS_CPU_BUILTIN_MEMCPY_MIN  = 0,
```

Comment:
use struct instead of array of 2 entries

Reply:
done
force-pushed changes as the PR is small enough